website/content/docs/commands/var/list.mdx

```diff
@@ -0,0 +1,137 @@
+  results. Queries using this option are less efficient than using the prefix
+  parameter; therefore, the prefix parameter should be used whenever possible.
+
+- `-out` `(enum: go-template | json | table | terse )`: Format to render the
+  variable in. When using "go-template", you must provide the template content
+  with the `-template` option. Defaults to "table" when stdout is a terminal and
+  to "json" when stdout is redirected.
+
+- `-template` `(string: "")` Template to render output with. Required when
+  output is "go-template".
+
+## Examples
+
+List values under the key "nomad/jobs":
+
+```shell-session
+$ nomad var list nomad/jobs
+Namespace  Path                           Last Updated
+default    nomad/jobs/example             2022-08-23T10:35:47-04:00
+default    nomad/jobs/variable            2022-08-23T10:24:45-04:00
+default    nomad/jobs/variable/www        2022-08-23T10:24:45-04:00
+default    nomad/jobs/variable/www/nginx  2022-08-23T10:24:46-04:00
+```
+
+List values under the key "nomad/jobs/variable/www" in JSON format:
+
+```shell-session
+$ nomad var list -out=json -namespace="*" nomad/jobs/variable/www
+[
+  {
+    "Namespace": "default",
+    "Path": "nomad/jobs/variable/www",
+    "CreateIndex": 1457,
+    "ModifyIndex": 1457,
+    "CreateTime": 1662061225600373000,
+    "ModifyTime": 1662061225600373000
+  },
+  {
+    "Namespace": "default",
+    "Path": "nomad/jobs/variable/www/nginx",
+    "CreateIndex": 800,
+    "ModifyIndex": 1000,
+    "CreateTime": 1662061717905426000,
+    "ModifyTime": 1662062162982630000
+  }
+]
+```
+
+Perform a paginated query:
+
+```shell-session
+$ nomad var list -per-page=3
+Namespace  Path                           Last Updated
+default    nomad/jobs/example             2022-08-23T10:35:47-04:00
+default    nomad/jobs/variable            2022-08-23T10:24:45-04:00
+default    nomad/jobs/variable/www        2022-08-23T10:24:45-04:00
+Next page token: default.nomad/jobs/variable/www/nginx
+```
+
+To fetch the next page :
+
+```shell-session
+$ nomad var list -per-page=3 \
+  -page-token=default.nomad/jobs/variable/www/nginx
+Namespace  Path                 Last Updated
+default    nomad/jobs/variable/www/nginx  2022-08-23T10:24:46-04:00
+```
+
+Perform a paginated query with JSON formatting:
+
+```shell-session
+$ nomad var list -out=json -namespace="*" -per-page=1 nomad/jobs/variable/www
+{
+    "Data": [
+        {
+           "Namespace": "default",
+            "Path": "nomad/jobs/variable/www",
+            "CreateIndex": 1457,
+            "ModifyIndex": 1457,
+            "CreateTime": 1662061225600373000,
+            "ModifyTime": 1662061225600373000
+        }
+    ],
+    "QueryMeta": {
+        "KnownLeader": true,
+        "LastContact": 0,
+        "LastIndex": 43,
+        "NextToken": "default.nomad/jobs/variable/www/nginx",
+        "RequestTime": 875792
+    }
+}
+```
+
+As with the tabular version, provide the `QueryMeta.NextToken` value as the
```

Comment:
```suggestion
As with the tabular version, provide the
```

I don't think you need this part of the sentence, but ignore if you do.

Reply:
Yeah, it's telling you where to get the value that you pass into `-page-token`.